### PR TITLE
:ambulance: wrong debt sign

### PR DESCRIPTION
**Pull request by KolushovAlexandr**

### CODE REVIEW

File: pos_debt_notebook/static/src/xml/pos.xml [hunk 1]
```diff
@@ -126,13 +126,14 @@
...
     <t t-name="CreditNote">
+        <!--PaymentMethod-->
         <t t-if="debt != 0">
             <t t-if="widget.pos.config.debt_type == 'debt'">
                 <span
                     t-attf-class="client-debt #{(widget.pos.get_client() and debt gt 0) ? 'negative' : ''} #{(widget.pos.get_client() and debt lt 0) ? 'positive' : ''}"
                 >
                     <t
-                        t-esc="(widget.pos.get_client() ? ' [Debt: ' + widget.format_currency(debt) + ']': '')"
+                        t-esc="(widget.pos.get_client() ? ' [Debt: ' + widget.format_currency(- debt) + ']': '')"
```
Comment (yelizariev) [inline pos_debt_notebook/static/src/xml/pos.xml:137]: Are you sure?

Comment (KolushovAlexandr) [reply pos_debt_notebook/static/src/xml/pos.xml:137]: yes i am

Comment (yelizariev) [reply pos_debt_notebook/static/src/xml/pos.xml:137]: ok, then it's the variable name which is confusing

File: pos_debt_notebook/static/src/xml/pos.xml [hunk 2]
```diff
@@ -127,6 +127,7 @@
 
     <t t-name="CreditNote">
         <!--PaymentMethod-->
+        <!-- TODO: rename 'debt' variable to something matching -->
```
Comment (yelizariev) [inline pos_debt_notebook/static/src/xml/pos.xml:130]: :+1:

**Review by Ramil-Mukhametzyanov — APPROVED** [review]

**Comment by ilmir-k** [comment]
I approve to merge it now

**Comment by trojikman** [comment]
I approve to merge it now

**Pull request merged**